### DISCUSSION TITLE
Port to aquila printers?

### DISCUSSION THREAD

**Discussion by knownchild** [mriscoc/Ender3V2S1#96] [Q&A]
I love this firmware, how can i use it on my voxelab aquila?

**Comment by mriscoc** [comment]
Hi, Alex has a great firmware for Voxelab Aquila: https://github.com/alexqzd/Marlin/releases
I don't have that printer and do not know anything about them. Maybe it is as simple as port the configuration files in the Professional Firmware.

**Reply by knownchild** [reply]
Alex firmware is good, but i like how this one works, much much better. The bed leveling alone is super fast

**Comment by Korgoth1234** [comment]
Hello,
 
I tried changing the ini and buildroot directories, and the "platformio.ini" file from another firmware: 
https://github.com/Jyers/Marlin/releases/tag/v2.0.1

But it didn't work, the firmware is build, but it won't flash into the printer.
I can test the firmware works with Voxelab Aquila, but I can't understand the difference of these two firmware in hardware.

**Reply by knownchild** [reply]
This is what i was trying to do with this firmware, but i am unsure what files to edit exactly. total noob compiling firmware, i was able to edit firmware with autobuildMarlin plugin.
Figured someone way more knowledgeable than i could port this over to an Aquila and maybe other ender 3 clones.
